db/migrate/20150915001329_assign_tenant_to_miq_request.rb

```diff
@@ -0,0 +1,71 @@
+class AssignTenantToMiqRequest < ActiveRecord::Migration
+  class Tenant < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    # seed and return the current root_tenant
+    def self.root_tenant
+      Tenant.where(:ancestry => nil).first || Tenant.create!(:use_config_for_attributes => true)
+    end
+  end
+
+  class MiqAeNamespace < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class Provider < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class TenantQuota < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class Vm < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqRequest < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class Service < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ServiceTemplateCatalog < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def change
+    models = [ExtManagementSystem, MiqAeNamespace, MiqGroup,
+              Provider, TenantQuota, Vm,
```

Comment:
Don't need TenantQuota here or it's migration stub since it already has a belongs_to tenant and `Tenant#set_quotas` handles creating a quota attached to a tenant.

Why Vm, not VmOrTemplate?  I think we want to set tenant on templates too, right?